Source/Validator/StateValidator.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+class StateValidator: Validator {
+    func validate(text: String?) -> Bool {
+        return isEmpty(text: text)
```

Comment:
Implicit return is possible. Is this final version, a StateValidator just checks text exists?

Reply:
For now, Yes.